models/enterprise_sonic/snmp/deleted_example_02.txt

```diff
@@ -1,23 +1,30 @@
 # Using deleted
-
-# Before state:
-# -------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# Before state:
+#---------------
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# show running-configuration | grep snmp
 #
+# snmp-server user user1 group group-lab encrypted auth
+# md5 auth-password
+# U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+# priv aes-128 priv-password U2FsdGVkX1/Xs+ffZvdV9YzfyGHgIJ+zkLRPfF3/WgYIE1S4Ribvbzhu5chpHHI7ooCBpcVxYZotAXDzgetxvQ==
+# snmp-server group group-floor2 v3 priv read r_view write w_view notify n_view
+# snmp-server community comm1 group group-lab
+# snmp-server user user2 group group-lab
+# auth sha auth-password U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+# priv aes-128 priv-password U2FsdGVkX1/Xs+ffZvdV9YzfyGHgIJ+zkLRPfF3/WgYIE1S4Ribvbzhu5chpHHI7ooCBpcVxYZotAXDzgetxvQ==
 
-- name: Delete all SNMP communities
+- name: Delete all SNMP server configuration
```

Comment:
Please clarify that the maximum scope of deletion is a single top level sub-option.

```suggestion
- name: Delete all SNMP server configuration for the "user" sub-option.
```

Reply:
Verified.